specification/_global/delete_by_query/DeleteByQueryRequest.ts

```diff
@@ -56,18 +56,18 @@ export interface Request extends RequestBase {
     preference?: string
     refresh?: boolean
     request_cache?: boolean
-    requests_per_second?: long
+    requests_per_second?: float
```

Comment:
Should `requests_per_second` be `long|float` instead of only `float`?

Reply:
ES really expects a float here, which from a JSON perspective also includes integer values (i.e. no decimal point).

[The docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html) are a bit misleading through, saying "you can set requests_per_second to any positive decimal number" and later "requests_per_second (Optional, integer)"...